src/components/TextInput/TextInput.tsx

```diff
@@ -18,17 +18,17 @@ interface Props {
   placeholder?: string
   /** Defines whether the input should be disabled or not. Defaults to false. */
   disabled?: boolean
+  /** Defines the custom error message of the input. */
+  isInvalidFeedback?: string
   /** Defines whether the input should display as invalid. Defaults to false. */
   isInvalid?: boolean
   /** Defines whether the input should display as valid. Defaults to false */
-  isValid?: boolean
-  /**
-   * Defines the class of the textinput.
-   */
+  isValid?: boolean | true
```

Comment:
no reason to have `boolean | true` since the `true` part is redundant. We can replace this simply with `isValid?: boolean`